packages/amplify-schema-validator/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "strict": true,
+    "sourceMap": true,
+    "esModuleInterop": true,
+    "declaration": true,
+    "outDir": "./dist"
+  },
+  "include": ["src"],
+  "exclude": ["src/__tests__"]
+}
```

Comment:
Can't open the yarn.lock file so adding the comment here. Did you run `yarn setup-dev`? If it is so, we don't check in that version of yarn.lock file. (I'm think we haven't documented that clearly in contribution page).

I don't think yarn file should contain these many changes considering the number of new dependencies added. Can you do `yarn clean && yarn` and give a try?

Reply:
I did re-run `yarn` per @alharris-at feedback. I just now ran `yarn clean && yarn` and there is no diff.